src/Microsoft.AspNetCore.Sockets/ConnectionManager.cs

```diff
@@ -96,8 +96,6 @@ public void Scan()
                 return;
             }
 
-            _logger.ScanningConnections();
```

Comment:
Are we going to log when a connection is disposed as part of this change or this is separate?